src/Components/Components/test/Auth/AuthorizeViewTest.cs

```diff
@@ -331,15 +330,9 @@ public void RendersAuthorizingUntilAuthorizationCompleted()
             Assert.Equal(2, renderer.Batches.Count);
             var batch2 = renderer.Batches[1];
             var diff2 = batch2.DiffsByComponentId[authorizeViewComponentId].Single();
-            Assert.Collection(diff2.Edits,
-            edit =>
-            {
-                Assert.Equal(RenderTreeEditType.RemoveFrame, edit.Type);
-                Assert.Equal(0, edit.SiblingIndex);
-            },
-            edit =>
+            Assert.Collection(diff2.Edits, edit =>
             {
-                Assert.Equal(RenderTreeEditType.PrependFrame, edit.Type);
+                Assert.Equal(RenderTreeEditType.UpdateText, edit.Type);
```

Comment:
The diff for `AuthorizeView` got more efficient (it now updates text in place, instead of removing and re-adding) because of the change I made to the sequence numbers. This was discovered when writing unit tests for `AuthorizeRouteView`.

Reply:
nice